src/app/pages/news/9.4/12-cms.md

```diff
@@ -1,16 +1,15 @@
 ## CMS {#cms}
 
-User experience is a top priority in process automation projects. 
-That is why the Axon Ivy Portal has been massively enhanced.
+The content management system (CMS for short) was completely renovated. The internal concept was radically changed resulting in a smart CMS with many new features.
```

Comment:
past --> past perfect tense: was --> has been.  the renovation has been completed, thus it was no longer running.